src/main/java/edu/cornell/kfs/tax/batch/dto/SprintaxInfo1042S.java

```diff
@@ -1,11 +1,114 @@
 package edu.cornell.kfs.tax.batch.dto;
 
-/**
- * This object will be fully implemented by one of the follow-up Sprintax user stories.
- */
+import java.sql.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.krad.bo.BusinessObject;
+
+import edu.cornell.kfs.tax.batch.dataaccess.TaxDtoFieldEnum;
+
 public class SprintaxInfo1042S {
 
+    private String rowId;
+    private String taxId;
+    private String payeeId;
+    private Integer vendorHeaderId;
```

Comment:
I'm confused to what this class is representing by looking at the name. Since it is holding the Vendor information should it be named Sprintax1042SVendorInfo? Do I have the purpose correct?

Reply:
This class will eventually hold the information that's needed for generating the file lines in the Sprintax biographic and payment files. Many of these fields come from my earlier all-in-one Sprintax PR that was later abandoned, so most of the code that actually uses these fields won't be in place until Part 4 is implemented.

I'll remove the unused fields for now (and possibly rename the class), and will add things back in for the Part 4 work.